sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
-structure [@tag:Kearnes2016_admet], low parameter networks based on chemical
-similarity (IRV) work very well, especially well-suited for the domain in which
-training data can be limited and contains few positive instances, may touch on
-BACE example here and other discussions of training data limitations (e.g.
-[@tag:AltaeTran2016_one_shot])
-- "Creative experimentation" phase of the field, new ideas for representation
-learning and novel approaches including graph convolutions, autoencoders,
-one shot learning, and generative models
-- These "creative" approaches are definitely interesting but aren't necessarily
-outperforming existing methods, improvements on the software and
-reusability side could be important to help establish more rigorous
-benchmarking, DeepChem as example of this
-- Future outlook, what would need to happen for the "creative" approaches
-to overtake the current state of the art, can representation learning be
-improved by incorporating more information about chemical properties or
-even more "tasks" during training, how much will future growth depend on
-data versus algorithms
-- Future outlook part 2, how the above approaches relate to traditional
-methods like docking (note neural networks that include docking scores as
-features), deep learning efforts in this direction that use structure (e.g.
-[@tag:Wallach2015_atom_net @arxiv:1612.02751]), "zero-shot learning",
-analogies to other domains where deep learning can capture the behavior
-of complex physics (e.g. quantum physics example), maybe briefly mention
-other compound-protein interaction-based networks although that doesn't seem
-to fit here and is somewhat out of scope
-- Future output part 3 (most speculative), what would successful generative
-networks mean for the HTS field?
+In the biomedical domain, high-throughput chemical screening aims to improve
+therapeutic options over a long term horizon [@tag:PerezSianes2016_screening]
+`TODO: add another general screening reference`.  The objective is to discover
+which small molecules (also referred to as chemical compounds or ligands)
+effectively and specifically affect the activity of a target, such as a kinase,
+protein-protein interaction, or broader cellular phenotype.  This screening
+process can serve as the first step in the long drug discovery pipeline, where
+novel chemicals are pursued for their ability to inhibit or enhance
+disease-relevant biological mechanisms.  The appeal of machine learning in this
+domain is the need to improve the efficiency of the initial high-throughput
+screens such that sufficient candidate active compounds can be identified
+without exhaustively screening libraries of hundreds of thousands or millions of
+chemicals.  This task has been treated as a classification, regression, and
+ranking problem.  In reality, it does not fit neatly into any of those
+categories.  An ideal algorithm will rank a sufficient number of active
```

Comment:
Is sufficient target dependent, or does it have a general meaning in this context (i.e. 10 lead compounds should be enough for anybody).